pyuvdata/utils.py

```diff
@@ -1490,6 +1499,8 @@ def uvcalibrate(
     Dterm_cal=False,
     delay_convention="minus",
     undo=False,
+    override_time_check=False,
```

Comment:
does it make more sense to just make this `time_check = True` and `ant_check = True`? I think this may be more consistent with extra checks in uvdata, but I may be wrong.

Reply:
hmm, that might make more sense, let's let @adampbeardsley weigh in.

Reply:
Honestly I don't think it makes a big difference. I'd slightly lean toward Nick's suggestion just because it simplifies variable names and feels less like a double negative.

Reply:
ok.